docker/suricata/dist/suricata.yaml

```diff
@@ -57,7 +59,7 @@ default-log-dir: /var/log/suricata/
 
 # global stats configuration
 stats:
-  enabled: no
+  enabled: yes
```

Comment:
I enabled this because of the error below in my logs, but we can disable it altogether if not interested:
```
<Error> - [ERRCODE: SC_ERR_STATS_LOG_GENERIC(278)] - eve.stats: stats are disabled globally: set stats.enabled to true.

Reply:
If nobody is using it, we could consider disabling it here:
https://github.com/telekom-security/tpotce/blob/e2f76c44cbf10664099d5d336848dbf4dd065562/docker/suricata/dist/suricata.yaml#L278-L281

Reply:
Good idea!